src/common/istore.d.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// tslint:disable-next-line:interface-name
-export interface IBaseStore<TState> {
+
+export interface BaseStore<TState> {
```

Comment:
should we change the file name here? and move from `.d.ts` to `.ts` is a good idea too

Reply:
I want to, but  again only if It doesn't increase the number of changes too much!

Reply:
Done. Changed it to `base-store.ts`